modules/standard/Map.chpl

```diff
@@ -87,9 +87,13 @@ module Map {
        than 50% full. The acceptable values for this argument are
        between 0 and 1, exclusive, meaning (0,1). This is useful
        when you would like to reduce memory impact or potentially
-       speed up how fast the map finds a slot.
+       speed up how fast the map finds a slot. To override the
+       default value of 0.5, the `defaultHashTableResizeThreshold`
```

Comment:
I think you might be able to set this up so that it can link to section you added to the spec